src/rgw/rgw_rados.h

```diff
@@ -2595,10 +2582,6 @@ class RGWRados
 
   int create_pool(const rgw_pool& pool);
 
-  /**
-   * create a bucket with name bucket and the given list of attrs
-   * returns 0 on success, -ERR# otherwise.
-   */
   int init_bucket_index(RGWBucketInfo& bucket_info, int num_shards);
```

Comment:
I think it would be better to move this comment to create_bucket() declaration in the same file, instead of dropping it.